autotest/test_prms_sro.py

```diff
@@ -0,0 +1,40 @@
+import os
+import sys
+
+import pytest
+
+cwd = os.getcwd()
+if cwd.endswith("autotest"):
+    sys.path.append("..")
+    rel_path = ".."
+elif cwd.endswith("pynhm"):
+    sys.path.append(".")
+    rel_path = "."
+
+import pynhm
+
+base_path = os.path.join(rel_path, "test_data")
+
+parameter_file_dict = {
+    "box_01": os.path.join(base_path, "box_01", "box_01.params"),
+}
+
+
+@pytest.mark.parametrize("domain_key", parameter_file_dict.keys())
+def test_prms_surface_runoff(domain_key):
```

Comment:
use the domain fixture now and you can get rid of base_path and parameter_file_dict
though, for the moment, box_01.yaml dosent have enough fields to currently pass through conftest. this is on my list of things to address. We can make fields optional in conftest if necessary

Reply:
It would be very good to have two tests to run in CI (drb_2yr and box_01) to ensure all tests run on multiple domains.